mailpoet/assets/css/src/components-automation-editor/_dropdown.scss

```diff
@@ -1,3 +1,39 @@
+.mailpoet-automation-editor-edit-site-document-actions {
```

Comment:
@JanJakes, I'll update CSS  so that the result is closer to the original.

> Here I'm wondering:
> 
> Should we use the new look to align with Gutenberg, or it doesn't work in this case?
> Should we create our custom component for this?

Good questions. 

The new Gutenberg design is something completely different. Click on the component triggers the command palette.

<img width="537" alt="Screenshot 2024-01-19 at 15 26 08" src="https://github.com/mailpoet/mailpoet/assets/1082140/38031022-0ec4-4e25-9a28-496b7a50049f">

As I see it, what we currently have is not aligned with Gutenberg (design or purpose), so we should create a custom component. Or even discuss with the design team if we still want to use that area (center of the header) for this component as it is now not aligned with Gutenberg.

Do you want me to create the component within this PR? I would suggest going with the CSS fix for now so that we no longer block this PR and I would prepare a ticket about refactoring the component. Would you be ok with that?